Geometry/RPCGeometryBuilder/plugins/RPCGeometryValidate.cc

```diff
@@ -101,16 +101,17 @@ class RPCGeometryValidate : public one::EDAnalyzer<> {
 };
 
 RPCGeometryValidate::RPCGeometryValidate(const edm::ParameterSet& iConfig)
-    : infileName_(iConfig.getUntrackedParameter<string>("infileName", "cmsGeom10.root")),
+    : tokRPC_{esConsumes<RPCGeometry, MuonGeometryRecord>(edm::ESInputTag{})},
+      infileName_(iConfig.getUntrackedParameter<string>("infileName", "cmsGeom10.root")),
       outfileName_(iConfig.getUntrackedParameter<string>("outfileName", "validateRPCGeometry.root")),
       tolerance_(iConfig.getUntrackedParameter<int>("tolerance", 6)) {
   fwGeometry_.loadMap(infileName_.c_str());
   outFile_ = new TFile(outfileName_.c_str(), "RECREATE");
 }
 
 void RPCGeometryValidate::analyze(const edm::Event& event, const edm::EventSetup& eventSetup) {
-  eventSetup.get<MuonGeometryRecord>().get(rpcGeometry_);
-  if (rpcGeometry_.isValid()) {
+  rpcGeometry_ = &eventSetup.getData(tokRPC_);
+  if (rpcGeometry_ != nullptr) {
```

Comment:
These lines change functionality. See this comment:
https://github.com/cms-sw/cmssw/pull/34214#discussion_r656695574